modules/dqt/ajax/DeleteDoc.php

```diff
@@ -0,0 +1,33 @@
+<?php
```

Comment:
This would allow any logged in user to delete any document in CouchDB regardless of permissions or document "type".

Reply:
Good catch, I have added the permission check.